.storybook/preview.tsx

```diff
@@ -54,6 +55,7 @@ export const decorators = [
     // Make sure the msw service worker is started, and reset the handlers to
     // defaults.
     (Story: any) => {
+        useFeatures([])
```

Comment:
Makes sure that the list of EE features used is `[]` before the beginning of each scene. Otherwise if a story calls this, it'll remain fixed until you open another story that calls `setFeatures`

Reply:
Would be super useful as a comment inline, for posterity